src/test/java/io/strimzi/test/systemtest/AbstractST.java

```diff
@@ -0,0 +1,69 @@
+package io.strimzi.test.systemtest;
+
+import io.strimzi.utils.StUtils;
+import io.strimzi.utils.k8s.KubeClusterResource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Stack;
+import java.util.stream.Collectors;
+
+import static io.strimzi.utils.k8s.KubeClusterResource.cmdKubeClient;
+import static io.strimzi.utils.k8s.KubeClusterResource.kubeClient;
+
+public class AbstractST {
+
+    private static final String USER_PATH = System.getProperty("user.dir");
+    private static final Logger LOGGER = LoggerFactory.getLogger(AbstractST.class);
+
+    public static final String NAMESPACE = "strimzi-drain-cleaner";
+    public static final String DEPLOYMENT_NAME = "strimzi-drain-cleaner";
+
+    protected static KubeClusterResource cluster;
+    private static Stack<String> createdFiles = new Stack<>();
+
+    static void applyInstallFiles() {
+        String installDirPath = cluster.isNotKubernetes() ? USER_PATH + "/install/openshift/" : USER_PATH + "/install/kubernetes/";
```

Comment:
Do we care about this? IMHO the kubernetes installation files should work even in OpenShift and it might simplify the code (not needing to care what the cluster on the other side is etc.).

Reply:
If the installation files works on both, I'll simplify it. Thanks :)